### PR TITLE
Logging fixes and c value fixes 

### PR DESCRIPTION
**Pull request by zachmanc**
- Only log errors if the `debug` flag is set on the ComScoreConfiguration
- Set c3, c4, c6 values to the correct value from the metadata 
- Change the default value of c3, c4, c6 to `*null` per documentation

### CODE REVIEW

File: src/ComScoreStreamingAnalytics.ts [hunk 1]
```diff
@@ -15,15 +13,23 @@ export class ComScoreStreamingAnalytics {
   private adBreakScheduleTime?: number;
   private logger: ComScoreLogger;
 
-  constructor(player: PlayerAPI, metadata: ComScoreMetadata = { mediaType: ComScoreMediaType.Other },
+  constructor(player: PlayerAPI, metadata: ComScoreMetadata = new ComScoreMetadata(),
               configuration: ComScoreConfiguration) {
-    this.logger = new ComScoreLogger(configuration);
```
Comment (stonko1994) [inline src/ComScoreStreamingAnalytics.ts]: I would prefer the instance of the logger but no blocker.

File: src/ComScoreStreamingAnalytics.ts [hunk 2]
```diff
@@ -253,9 +258,9 @@ export class ComScoreMetadata {
   digitalAirdate?: string;
   tvAirdate?: string;
   stationTitle?: string;
-  c3?: string;
-  c4?: string;
-  c6?: string;
+  c3: string = '*null';
```
Comment (stonko1994) [inline src/ComScoreStreamingAnalytics.ts]: `*null` looks od 🤔 Intended?

Comment (zachmanc) [reply src/ComScoreStreamingAnalytics.ts]: yes unfortunately

File: src/ComScoreStreamingAnalytics.ts (hunk 1 above)
Comment (zachmanc) [reply src/ComScoreStreamingAnalytics.ts]: Just curious, why is this?

**Pull request merged**